lib/ansible/runner/credential/ibmcloud_credential.rb

```diff
@@ -0,0 +1,22 @@
+module Ansible
+  class Runner
+    class IbmCloudCredential < Credential
+      def self.auth_type
+        'ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::IbmCloudCredential'
```

Comment:
This credential will be used to SSH into a PowerVC provider right? I'm wondering if something more descriptive like `PowerVCCredential` or maybe `ImageImportSSHCredential` would be better?